docs/historic.md

```diff
@@ -34,31 +34,39 @@ res.to_pandas().head()
 | 2022-06-01 08:30:00+00:00 |         12 |     0 |     0 |
 
 
-## Example of multi period request for the year of 2023 from Jan 01
-### The below code is using an unmetered location. If using a metered location, it will consume 12 request.
+## Example of 12 month combined multi-period requests
+The below code is using an unmetered location. If using a metered location, it will consume 12 request.
```

Comment:
requests.

Reply:
Added the period.

Reply:
@billy-solcast  it's back to "12 request."

Reply:
Apologies, updated the header instead of within the sentence.
Updated.